ca_gazebo/models/camera/model.sdf

```diff
@@ -0,0 +1,51 @@
+<?xml version='1.0'?>
+<sdf version='1.6'>
+  <model name='camera'>
+    <static>true</static>
+    <pose>-1 0 2 0 1 0</pose>
+    <link name='april_camera_link'>
+      <visual name='visual'>
+        <geometry>
+        <mesh>
+          <uri>model://camera/meshes/camera.stl</uri>
+          <scale>2.5 2.5 2.5</scale>
+        </mesh>
+        </geometry>
+      </visual>
+      <sensor name='my_camera' type='camera'>
+        <pose>0.2 0 0 -1.577 0 0</pose>
+        <camera>
+          <save enabled="false">
+            <path>/tmp/camera_save</path>
+          </save>
+          <horizontal_fov>1.047</horizontal_fov> <!-- 188.46 degrees-->
+          <image>
+            <width>1920</width>
+            <height>1080</height>
+          </image>
+          <clip>
+            <near>0.1</near>
+            <far>100</far>
+          </clip>
+        </camera>
+        <always_on>1</always_on>
+        <update_rate>30</update_rate>
+
+          <plugin name="april_tag_camera_controller" filename="libgazebo_ros_camera.so">
+            <alwaysOn>true</alwaysOn>
+            <updateRate>0</updateRate>
+            <cameraName>april_tag_camera</cameraName>
+            <imageTopicName>image_raw</imageTopicName>
```

Comment:
Use a more descriptive topic name. The idea is to understand what is this topic for only reading its name.

Reply:
set to april_tag_camera_image_raw